### ISSUE TITLE
Full pfm_description text not showing completely for one preference key

### ISSUE THREAD

**Issue by apizz** [erikberglund/ProfileCreator#117]
Weird issue where I have near exact same description text between three different preferences and for two the description wraps to a second line just fine, but the third does not.

```
<dict>
	<key>pfm_description</key>
	<string>When set to 'true', also searches the System Keychain for certificates to sign profiles.</string>
	<key>pfm_name</key>
	<string>SigningCertificateSearchSystemKeychain</string>
	<key>pfm_title</key>
	<string>Include Certificates from System Keychain</string>
	<key>pfm_type</key>
	<string>boolean</string>
</dict>
<dict>
	<key>pfm_description</key>
	<string>When set to 'true', also includes available untrusted certificates for signing profiles.</string>
	<key>pfm_name</key>
	<string>SigningCertificateShowUntrusted</string>
	<key>pfm_title</key>
	<string>Show Untrusted Certificates</string>
	<key>pfm_type</key>
	<string>boolean</string>
</dict>
<dict>
	<key>pfm_description</key>
	<string>When set to 'true', also includes available expired certificates for signing profiles.</string>
	<key>pfm_name</key>
	<string>SigningCertificateShowExpired</string>
	<key>pfm_title</key>
	<string>Show Expired Certificates</string>
	<key>pfm_type</key>
	<string>boolean</string>
</dict>
```
The result in ProfileCreator:
![screen shot 2019-02-07 at 7 20 26 pm](https://user-images.githubusercontent.com/11789931/52451713-201fc400-2b0e-11e9-8fca-aa271b21c3a2.png)

When I add a separate line in the description, only then does the rest of the text appear:
```
<dict>
	<key>pfm_description</key>
	<string>When set to 'true', also searches the System Keychain for certificates to sign profiles.</string>
	<key>pfm_name</key>
	<string>SigningCertificateSearchSystemKeychain</string>
	<key>pfm_title</key>
	<string>Include Certificates from System Keychain</string>
	<key>pfm_type</key>
	<string>boolean</string>
</dict>
<dict>
	<key>pfm_description</key>
	<string>When set to 'true', also includes available untrusted certificates for signing profiles.</string>
	<key>pfm_name</key>
	<string>SigningCertificateShowUntrusted</string>
	<key>pfm_title</key>
	<string>Show Untrusted Certificates</string>
	<key>pfm_type</key>
	<string>boolean</string>
</dict>
<dict>
	<key>pfm_description</key>
	<string>When set to 'true', also includes available expired certificates for 
signing profiles.</string>
	<key>pfm_name</key>
	<string>SigningCertificateShowExpired</string>
	<key>pfm_title</key>
	<string>Show Expired Certificates</string>
	<key>pfm_type</key>
	<string>boolean</string>
</dict>
```
![screen shot 2019-02-07 at 7 26 36 pm](https://user-images.githubusercontent.com/11789931/52451754-4c3b4500-2b0e-11e9-8d4b-246b3a92ecda.png)

**Comment by apizz** [comment]
Ran into another situation of this, albeit the description is much longer

![Screen Shot 2019-03-27 at 7 55 08 PM](https://user-images.githubusercontent.com/11789931/55119966-5c14e580-50ca-11e9-9b80-77b1c4b83190.png)

**Comment by apizz** [comment]
This is resolved in latest 0.2.5 beta build

**Comment by apizz** [comment]
Fixed in 0.2.5 release
